pantheon/src/main/java/tech/pegasys/pantheon/controller/PantheonControllerBuilder.java

```diff
@@ -199,6 +203,31 @@
 
     final MutableBlockchain blockchain = protocolContext.getBlockchain();
 
+    final Pruner pruner =
+        new Pruner(
+            new MarkSweepPruner(
+                protocolContext.getWorldStateArchive().getWorldStateStorage(),
+                storageProvider.createPruningStorage(),
+                metricsSystem),
+            protocolContext.getBlockchain(),
+            () ->
```

Comment:
I don't think we need to wrap this in an interface.  Especially since stop is only called at JVM shutdown.